libevm/tooling/release/cherrypick.sh

```diff
@@ -27,7 +27,7 @@ set -o pipefail;
 
 SELF_DIR=$(dirname "${0}")
 # The format of the `cherrypicks` file is guaranteed by a test so we can use simple parsing here.
-CHERRY_PICKS=$(< "${SELF_DIR}/cherrypicks" grep -Pv "^#" | awk '{print $1}')
+CHERRY_PICKS=$(< "${SELF_DIR}/cherrypicks" grep -Ev "^#" | awk '{print $1}')
```

Comment:
`-P` does not work on the OSX native grep

Reply:
I think everyone knows how I feel about OSX "tools" (the binaries, not the people who decided to not just use GNU).